contracts/test-utils/transfer-wrapper/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "transfer-wrapper"
+version = "0.1.0"
+authors = ["Victor Lopez <victor@dusk.network>"]
+edition = "2018"
+
+[dependencies]
+canonical = "0.6"
+canonical_derive = "0.6"
+dusk-abi = "0.9.0-rc"
+rusk-abi = { path = "../../../rusk-abi" }
+dusk-bytes = "0.1"
+dusk-bls12_381 = { version = "0.8", default-features = false, features = ["canon"] }
+dusk-jubjub = { version = "0.10", default-features = false, features = ["canon"] }
+dusk-poseidon = { version = "0.21.0-rc", default-features = false, features = ["canon"] }
```

Comment:
We're landing a bunch of rc's, so at least for the coming PR that are still not merged, we should point to the proper version, without rc.

Reply:
This crate contains the latest r3elease for all deps